lib/src/ble/ble_scanner.dart

```diff
@@ -29,6 +30,10 @@ class BleScanner extends StatefulStream<BleScanState> {
       addStateToStream(state);
     }, onError: (Object e) {});
     addStateToStream(state);
+
+    Future.delayed(_scanDuration, () {
+      stopScan();
```

Comment:
Stop scan here doesn't call `Wakelock.disable();`. And I'm not sure if  stop after 10sec is needed

Reply:
Most of tools, which scan, have some timout

Reply:
I will think how to resolve wakelock call

Reply:
Hmm, interesting what the reason

Reply:
fixed